pkg/operator/operator.go

```diff
@@ -1022,10 +1027,9 @@ func (o *Operator) Config(ctx context.Context, key string) (*manifests.Config, e
 	if *c.ClusterMonitoringConfiguration.UserWorkloadEnabled {
 		c.UserWorkloadConfiguration, err = o.loadUserWorkloadConfig(ctx)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("%w: %w", ErrInvalidUWMConfiguration, err)
 		}
 	}
-	o.userWorkloadEnabled = *c.ClusterMonitoringConfiguration.UserWorkloadEnabled
```

Comment:
wasn't used.

Reply:
A remnant of https://github.com/openshift/cluster-monitoring-operator/pull/2048 I think. Good catch!